utils/Utils.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+
+/**
+ * This class includes the utitity functions.
+ * @class
+ * @classdesc Provides the common utility functions.
```

Comment:
Not required as this is not a JSDoc for a constructor function that needs a separate description for the class.